index.html

```diff
@@ -39,6 +39,7 @@ <h2>I’m a software developer</h2>
             <h1>My Recent work</h1>
     <div class="allw">
```

Comment:
- Good Job using inline comments to highlight lines of code you are suggesting changes on. 💪🏽 

- I'd like to kindly suggest that it will be better to include an explanation, a "why", for each requested change. For example ("  I think it would be better to use section tech instead of div because a div . .. . .... ." )

- Also, i believe it is important to include some emoji's so as to make your comment sound more friendly. 🤗

Reply:
-Good Job using inline comments to highlight lines of code you are suggesting changes on. 💪🏽 

- I'd like to kindly suggest that it will be better to include an explanation, a "why", for each requested change. For example ("  I think it would be better to use section tech instead of div because a div . .. . .... ." )

- Also, it is important to also include some emoji's so as to make your comment sound more friendly. 🤗

Reply:
- Good Job using Inline comments in highlighting lines of code you are requesting changes on. 💪🏽 

- I think it is important to state reasons "WHY" a change needs to be implemented. Why don't we re phrase your comment to look something like this 👉🏽 (" I think it would be better to use section tech instead of div because section tag will . .... . . . . ") 

- Also, it will be great to include some emoji's in your comments in order to sound a bit more friendly. 🌸 

What do you think about this??